writer.go

```diff
@@ -7,6 +8,8 @@ import (
 	"os"
 	"sync"
 	"time"
+
+	"github.com/nsf/termbox-go"
```

Comment:
I'd like to avoid bringing in a dependency of this size. Will be great if you could please copy the relevent code and place it under vendor directory (including the licence)